helm/sidekiq-prometheus-exporter/values.yaml

```diff
@@ -97,6 +97,10 @@ containerPort: 9292
 # securityContext:
 #   runAsUser: 65534
 
+## Metrics exporter pod extra labels
+##
+# podExtraLabels:
```

Comment:
It would be cool if you can add `podAnnotations` and `podLabels` to the readme table 👍🏼